style/ruby/README.md

```diff
@@ -3,12 +3,11 @@ Ruby
 
 [Sample](sample.rb)
 
-* Avoid conditional modifiers (lines that end with conditionals).
```

Comment:
I think conditional modifiers, in general, can have value if used sensitively and judiciously. I'd like to empower devs to use them where they think they add value, so I'd like to remove the "Avoid..." and leave it to context.